movies/src/test/java/movies/CustomerTest.java

```diff
@@ -0,0 +1,44 @@
+package movies;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class CustomerTest {
+
+	@Test
+	public void customerCanBeCreated() {
+		assertNotNull(new Customer("Customer Test"));
+	}
+	
+	@Test
+	public void customerCanBeCreatedWithNameParameter(){
+		Customer newCustomer = new Customer("Customer Test");
+		assertTrue(newCustomer instanceof Customer);
+	}
+	
+	@Test
+	public void customerNameShouldBeSameThanTheObjectCreated(){
+		Customer newCustomer = new Customer("Customer Test");
+		assertEquals("Customer Test", newCustomer.getName());
+	}
+	
+	@Test
+	public void movieNewReleaseAmmountShouldBeThreeWhenTheDaysToRentAreMenorOrEqualThanTwo(){
+		Customer newCustomer = new Customer("Customer Test");
+		Movie newMovie = new Movie("The Jungle Book", 1);
+		Rental newRental = new Rental(newMovie, 2);
+		double ammount = newCustomer.selectMovie(0, newRental)/2;
+		assertEquals(3, ammount, 0);
+	}
+	
+	@Test
+	public void movieReleaseAmmountShouldBeTwoWhenTheDaysToRentAreMenorOrEqualThanTwo(){
+		Customer newCustomer = new Customer("Customer Test");
```

Comment:
I you notice the preconditions for your tests, you are repeating the Customer instance.
please add a before method.
```java
private Customer customer;
@Before
public void setUp() {
    customer = new Customer("Customer Test");
}
```
Then you can re-use the variable customer.